crates/enarx-config/src/lib.rs

```diff
@@ -13,6 +13,41 @@ use std::{collections::HashMap, ops::Deref};
 use serde::{de::Error as _, Deserialize, Deserializer};
 use url::Url;
 
+/// Configuration file template
+pub const CONFIG_TEMPLATE: &str = r#"# Configuration for a WASI application in an Enarx Keep
+
+# Environment variables
+# env = { "VAR1" = "var1", "VAR2" = "var2" }
```

Comment:
can we split these in multiple lines?
If we provide this as a template, we should also encourage good style. Having people put all their env variables on one line in a struct would be a maintenance nightmare

Reply:
done